scalahost/sbt/src/sbt-test/sbt-scalahost/semantic-example-slim/test

```diff
@@ -0,0 +1 @@
+> app/run
```

Comment:
It tests that the roundtrip produces a db that looks reasonable and has reasonable number of entries. Can you follow-up on this in a subsequent PR?